cvat/apps/documentation/user_guide.md

```diff
@@ -362,7 +389,41 @@ Use shortcuts for undo/redo actions ``Ctrl+Z`` __/__  ``Ctrl+Shift+Z``/``Ctrl+Y`
 
 ![](static/documentation/images/image061.jpg)
 
+---
+__Fill Opacity slider__
+
+Changes opacity of every Bounding Box in the Annotation.
```

Comment:
Change

Reply:
Please use lower case for bounding box  and annotation in the documentation. I don't see any reason why it starts with upper case.